src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInAdminEmailAttributes.java

```diff
@@ -0,0 +1,84 @@
+package teammates.client.scripts;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.SanitizationHelper;
+import teammates.logic.core.AdminEmailsLogic;
+import teammates.storage.api.AdminEmailsDb;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.appengine.api.datastore.Text;
+
+public class DataMigrationForSanitizedDataInAdminEmailAttributes extends RemoteApiClient {
+    private static final boolean isPreview = true;
+    private AdminEmailsDb adminEmailsDb = new AdminEmailsDb();
+    private AdminEmailsLogic adminEmailsLogic = AdminEmailsLogic.inst();
+    private int numberOfAffectedEmails;
+    private int numberOfUpdatedEmails;
+
+    public static void main(String[] args) throws IOException {
+        DataMigrationForSanitizedDataInAdminEmailAttributes migrator =
+                new DataMigrationForSanitizedDataInAdminEmailAttributes();
+        migrator.doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        List<AdminEmailAttributes> allEmails = adminEmailsLogic.getAllAdminEmails();
+        numberOfAffectedEmails = 0;
+        numberOfUpdatedEmails = 0;
+        DataMigrationForSanitizedDataHelper.LoopHelper loopHelper =
+                new DataMigrationForSanitizedDataHelper.LoopHelper(100, "admin emails");
+        System.out.println("Running data migration for sanitization on admin emails...");
+        System.out.println("Preview: " + isPreview);
+        for (AdminEmailAttributes email : allEmails) {
+            loopHelper.recordLoop();
+            fixSanitizedDataForEmail(email);
+        }
+        System.out.println("There are/is " + loopHelper.getCount() + " email(s).");
+        System.out.println("There are/is " + numberOfAffectedEmails + " affected email(s).");
+        System.out.println(numberOfUpdatedEmails + " email(s) are/is successfully updated.");
```

Comment:
We seem to be using `System.out.println` this a lot. Extract a `println(String)` method, just to reduce verbosity of the code?

``` java
System.out.println("There are/is " + loopHelper.getCount() + " email(s).");
System.out.println("There are/is " + numberOfAffectedEmails + " affected email(s).");
System.out.println(numberOfUpdatedEmails + " email(s) are/is successfully updated.");
```

``` java
println("There are/is " + loopHelper.getCount() + " email(s)." + "\n" +
        "There are/is " + numberOfAffectedEmails + " affected email(s)." + "\n" +
        numberOfUpdatedEmails + " email(s) are/is successfully updated.";
```